test/dummy_test.go

```diff
@@ -0,0 +1,134 @@
+package test
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+
+	"github.com/rollkit/go-sequencing"
+)
+
+func TestSubmitRollupTransaction(t *testing.T) {
+	sequencer := NewDummySequencer()
+
+	// Define a test rollup ID and transaction
+	rollupId := []byte("test_rollup_id")
+	tx := []byte("test_transaction")
+
+	// Submit a transaction
+	req := sequencing.SubmitRollupTransactionRequest{
+		RollupId: rollupId,
+		Tx:       tx,
+	}
+	resp, err := sequencer.SubmitRollupTransaction(context.Background(), req)
+
+	// Assert no error
+	assert.NoError(t, err)
+	// Assert the transaction was successfully added to the queue
+	assert.NotNil(t, resp)
+	assert.Equal(t, rollupId, sequencer.RollupId)
```

Comment:
I don't understand why the sequencer stores the `rollupId` and then having this check.

Does the sequencer need to store the rollupId if we are submitting it as a param in the API? Storing it feels specific to a based sequencer.

Reply:
removed.

Reply:
moved to constructor